spec/features/region_admin/shiftless_volunteers_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'Region Admin Shiftless Volunteers' do
+  feature 'Viewing shiftless volunteers' do
+    let(:boulder) { create(:region, name: 'Boulder') }
+    let(:denver) { create(:region, name: 'Denver') }
+
+    context 'as a region admin' do
+      let(:current_volunteer) { create(:volunteer, regions: [], assigned: true) }
+      let(:shiftless_volunteer) { create(:volunteer, regions: [boulder], assigned: true) }
+      let(:shiftless_volunteer_denver) { create(:volunteer, regions: [denver], assigned: true) }
+      let(:volunteer_with_shifts) { create(:volunteer, regions: [boulder], assigned: true) }
+      let(:schedule_chain) { create(:schedule_chain, region: boulder) }
+
+      before do
+        create(:assignment, :admin, volunteer: current_volunteer, region: boulder)
+        create(:assignment, volunteer: shiftless_volunteer, region: boulder)
```

Comment:
I don't have a full knowledge of the domain and what a shiftless volunteer would be... but wouldn't having an assignment create a shift for the volunteer?